static/app/views/performance/newTraceDetails/traceSearch/traceSearchEvaluator.spec.tsx

```diff
@@ -1,103 +1,17 @@
 import {waitFor} from 'sentry-test/reactTestingLibrary';
 
-import type {RawSpanType} from 'sentry/components/events/interfaces/spans/types';
-import type {TraceTree} from 'sentry/views/performance/newTraceDetails/traceModels/traceTree';
-import {TraceTreeNode} from 'sentry/views/performance/newTraceDetails/traceModels/traceTreeNode';
+import {TraceTree} from 'sentry/views/performance/newTraceDetails/traceModels/traceTree';
+import {
```

Comment:
The changes in this file is from:
- Actually using the `TraceTree` class methods instead of mocks
- Incrementing indices for callback checks by 1, since the tree class adds a root element at the top of the list.